api/server/response_options.proto

```diff
@@ -14,4 +24,11 @@ message ResponseOptions {
   uint32 response_body_size = 2 [(validate.rules).uint32 = {lte: 4194304}];
   // If true, then echo request headers in the response body.
   bool echo_request_headers = 3;
+
+  oneof oneof_delay_options {
```

Comment:
Is there a universal convention for whether file-based config settings have precedence over request header settings or vice versa? If not, could you document those here?

Reply:
good point, let me document that. The test-server extension assumes the static/file-based config is the default. The request-level config will get merged, and override. As these extensions share the same header / proto, I think they should share the same convention. SG?

Reply:
SGTM, thanks.